### ISSUE TITLE
Could not find the matching Reference-Protein in AHRD's database

### ISSUE THREAD

**Issue by jrtejero** [groupschoof/AHRD#17]
Hello everyone,

Once I managed to solve my problem with java memory (due to a bad RegExp writting) y am facing another persistent error. Apparently, the AHRD's database is unable to provide me information about my proteins, giving this error:

**_We are sorry, an un-expected ERROR occurred:
ahrd.exception.MissingAccessionException: Found Blast-Hits to reference protein 'P09406.1' but could not find the matching Reference-Protein in AHRD's persistent Database. Please update it accordingly.
	at ahrd.model.BlastResult.parseBlastDatabase(Unknown Source)
	at ahrd.model.BlastResult.readBlastResults(Unknown Source)
	at ahrd.controller.AHRD.parseBlastResults(Unknown Source)
	at ahrd.controller.AHRD.setup(Unknown Source)
	at ahrd.controller.AHRD.main(Unknown Source)_**

Could anybody help me?

Thanks in advance.

**Comment by sanjaysingh765** [comment]
I think AHRD is specifically designed to work with Swiss-Prot, tremble and TAIR only. You may be used your blast database other than it which may be the reason behind this error.

**Comment by jrtejero** [comment]
Dear sanjay,

I solved this problem months ago. The issue was related with an unexplainable mistake in the protein headers from the swissprot databes (probably, due to a failed downloading). Once I got the UNIPROTdatabase (downloaded fronm the official site: https://www.uniprot.org/downloads), a clean JAVA installation (I had to remove several older versions) and the last stable version of AHRD that I was told (https://github.com/groupschoof/AHRD/issues/16) I was able to annotate my data without any further problem.

Thanks for your help.

Regards,
